packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -349,27 +349,32 @@ export default (containerEl, environment) => {
             $rootScope.$emit('ovh::sidebar::hide');
             $state.go('app.error', { error });
           }
-        });
-
-        $state.defaultErrorHandler((error) => {
-          console.log(error);
           if (error.type === RejectType.ERROR && !error.handled) {
             $rootScope.$emit('ovh::sidebar::hide');
             $state.go(
               'error',
               {
                 detail: {
                   message: get(error.detail, 'data.message'),
+                  status: error.detail.status,
                   code: has(error.detail, 'headers')
                     ? error.detail.headers('x-ovh-queryId')
                     : null,
                 },
+                to: {
+                  state: transition.to(),
+                  params: transition.params(),
+                },
               },
               { location: false },
             );
           }
         });
 
+        $state.defaultErrorHandler((error) => {
+          console.log(error);
```

Comment:
is it intended that you have ```console.log``` as defaultErrorHandler here but in other apps it's ```return error```?

Reply:
If we don't have `defaultErrorHandler` in the app, `ui-router` will log the error to the console as a default behaviour. To retain the existing behaviour of not logging the error, I have included a `defaultErrorHandler` in all the apps and just return the error. In case of `dedicated` app, the error was logged to console. So I didn't modify and since the callback function wont be empty, I didn't add 'return error` statement.